lib/Github/Api/PullRequest.php

```diff
@@ -115,10 +115,18 @@ public function merged($username, $repository, $id)
         return $this->get('repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/pulls/'.rawurlencode($id).'/merge');
     }
 
-    public function merge($username, $repository, $id, $message, $sha)
+    public function merge($username, $repository, $id, $message, $sha, $squash = false, $title = null)
     {
-        return $this->put('repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/pulls/'.rawurlencode($id).'/merge', array(
-            'commit_message' => $message, 'sha' => $sha
-        ));
+        $params = array(
+            'commit_message' => $message,
+            'sha' => $sha,
+            'squash' => $squash,
```

Comment:
It would be safer to only pass through `squash` if it's set to true, otherwise omit it, wouldn't it?